package/govuk-prototype-kit.config.json

```diff
@@ -10,5 +10,120 @@
   ],
   "sass": [
     "/govuk/all.scss"
+  ],
+  "nunjucksComponents": [
```

Comment:
If we can't generate this programatically, it might be worth adding something to `/tasks/gulp/__tests__/after-build-package.test.js` to check that all components are in this list, and that the referenced `file` looks correct and exists.

That'll then hopefully prevent us from forgetting to e.g. add new components to the list.

Reply:
That's fair.  If that can be picked up by automated tests then it can be generated automatically too :)